config/jobs/kubernetes/sig-testing/kubernetes-kind-ci.yaml

```diff
@@ -100,55 +100,6 @@ periodics:
           # during the tests more like 3-20m is used
           cpu: 7
 
-- interval: 4h
-  cluster: k8s-infra-prow-build
-  name: ci-kubernetes-e2e-kind-alpha-features
-  annotations:
-    testgrid-dashboards: sig-release-master-informing, sig-testing-kind
-    testgrid-tab-name: kind-master-alpha
-    description: Runs tests with no special requirements other than alpha feature gates in a KinD cluster where alpha feature gates and APIs are enabled.
-    testgrid-alert-email: bentheelder@google.com,antonio.ojea.garcia@gmail.com,release-team@kubernetes.io
-    testgrid-num-columns-recent: '6'
-  labels:
-    preset-dind-enabled: "true"
-  decorate: true
-  decoration_config:
-    timeout: 60m
-  extra_refs:
-  - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
-  spec:
-    containers:
-    - image: gcr.io/k8s-staging-test-infra/krte:v20250904-c89b045f57-master
-      command:
-      - wrapper.sh
-      - bash
-      - -c
-      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
-      env:
-      - name: FEATURE_GATES
-        value: '{"AllAlpha":true,"EventedPLEG": false}'
-      - name: RUNTIME_CONFIG
-        value: '{"api/alpha":"true", "api/ga":"true"}'
-      - name: LABEL_FILTER
-        value: "Feature: isSubsetOf OffByDefault && !BetaOffByDefault && !Deprecated && !Slow && !Disruptive && !Flaky"
```

Comment:
we skip BetaOffByDefault here so that we do not need to remove this job

Reply:
The problem is not which tests we run, it is the cluster config itself (see https://github.com/kubernetes/kubernetes/pull/133697#issuecomment-3289638757 and following).